rbac/management/notifications/notification_hanlders.py

```diff
@@ -0,0 +1,179 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Notification handlers of object change."""
+import logging
+
+from django.conf import settings
+from management.notifications.producer_util import NotificationProducer
+
+from api.models import Tenant
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+producer = NotificationProducer()
+
+
+def notify_all(event_type, payload):
```

Comment:
Haven't done that performance-wise in rbac yet. Iterating through the tenants would be time consuming. And definitely impacting server if we run seeds there, so we might have to run seeds in worker like we did before.

I did ask about the issue but the notifications team said they could handle this work load, and Chris @wcmitchell  says Kafka should be fine with these amount of messages since they have millions of messages flying by.